### PR TITLE
[DIA-1735] linked no action and unknown values added

### PR DESCRIPTION
**Pull request by Nevazhnovu**

### CODE REVIEW

**Comment by carmelo-iriti** [comment]
it looks that this is the same MR of https://github.com/SourcePointUSA/android-cmp-app/pull/591/files

File: cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/network/model/ConsentRespExt.kt [hunk 1]
```diff
@@ -73,9 +73,10 @@ internal fun Map<String, Any?>.toCCPAUserConsent(uuid: String?, applies: Boolean
 
     val status: CcpaStatus = getFieldValue<String>("status")
         ?.let { s ->
-            CcpaStatus.values().find {
+            val status = CcpaStatus.values().find {
                 it.name == s
             }
+            return@let status ?: CcpaStatus.unknown
         }
         ?: fail("CCPAStatus cannot be null!!!")
```
Comment (carmelo-iriti) [inline cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/network/model/ConsentRespExt.kt:81]: this condition should be removed

**Review by carmelo-iriti — CHANGES_REQUESTED** [review]

File: cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/network/model/ConsentRespExt.kt [hunk 2]
```diff
@@ -76,9 +76,9 @@ internal fun Map<String, Any?>.toCCPAUserConsent(uuid: String?, applies: Boolean
             val status = CcpaStatus.values().find {
                 it.name == s
             }
-            return@let status ?: CcpaStatus.unknown
+            status ?: CcpaStatus.unknown
```
Comment (carmelo-iriti) [inline cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/network/model/ConsentRespExt.kt:79]: Let’s also verify if this extension is currently used in the code and if the functions that use it are also used. If they are belonging to the v6 and are not used in the v7 we can delete them

**Review by carmelo-iriti — CHANGES_REQUESTED** [review]

File: cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/network/model/ConsentRespExt.kt (hunk 2 above)
Comment (Nevazhnovu) [reply cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/network/model/ConsentRespExt.kt:79]: @carmelo-iriti this function is not used in v7 at all. It is used in v6 only.

Comment (carmelo-iriti) [reply cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/network/model/ConsentRespExt.kt:79]: In this case I would say that we can remove it and its all its flow

Comment (Nevazhnovu) [reply cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/network/model/ConsentRespExt.kt:79]: Wait a minute. It is used while changing `localStateStatus`

**Pull request closed without merging**